charts/woodpecker/charts/agent/values.yaml

```diff
@@ -49,19 +50,35 @@ extraVolumes: []
   #     claimName: example
 
 # -- Additional volumes that will be attached to the agent container
-extraVolumeMounts: []
+extraVolumeMounts:
+  []
   # - name: ca-certs
   #   mountPath: /etc/ssl/certs/ca-certificates.crt
 
 # -- Add additional init containers to the pod (evaluated as a template)
 initContainers: []
 
+persistence:
+  # -- Enable the creation of the persistent volume
+  enabled: true
+  # -- Defines an existing claim to use
+  existingClaim:
+  # -- Defines the size of the persistent volume
+  size: 1Gi
+  # -- Defines the path where the volume should be mounted
+  mountPath: '/etc/woodpecker'
+  # -- Defines the storageClass of the persistent volume
+  storageClass: ""
```

Comment:
There is a mix of single and double quotes used now. This PR has changes most double quotes to single, while this change seems to be unrelated to this PR. We should consider adding a yamllint config to enforce a style.